### ISSUE TITLE
all logfiles should contain the jobid in their filename

### ISSUE THREAD

**Issue by chrisdane** [esm-tools/esm_tools#842]
**Is your feature request related to a problem? Please describe.**
In an `awicm-1.0-recom` simulation, 5 logfiles are created in the `log/` directory per run:
```
<expid>_awicm_compute_<from>-<to>_<jobid>.log
<expid>_awicm_observe_compute_<from>-<to>.log
<expid>_awicm_prepcompute_<from>-<to>.log
<expid>_awicm_tidy_<from>-<to>.log
<expid>_esm_runscripts_<from>-<to>.log
```
Only one of them contains the jobid. If a specific date is computed more than once, the logfiles without the jobid in their filenames will be overwritten. 

**Describe the solution you'd like**
All logfiles should contain the jobid in their filename.

Thanks and cheers,
Chris

**Comment by mandresm** [comment]
Good catch. The good news is that the `compute` one contains all of them in one single file, so the information of the ones that are not stamped with the jobid is not lost.

**Comment by pgierz** [comment]
If we are opening this part of the runscripts anyway, may I suggest for the PR to be able to fully customise this, but with sensible defaults as suggested? I would say job type, date, slurm ID (maybe also Process ID)

If someone wants to customise and put their logs into blah123 then so be it. I'm happy to look, but I will guess @mandresm alreafy has the relevant parts open in his vim.
